docs/install/requirements.md

```diff
@@ -26,6 +26,7 @@ A three-node cluster is required to realize the multi-node features of Harvester
 - The first node always defaults to be a management node of the cluster.
 - When there are three or more nodes, the two other nodes added first are automatically promoted to management nodes to form a high availability (HA) cluster.
 - We recommend server-class hardware for the best results. Laptops and nested virtualization are not officially supported.
+- The Product_UUID (fetched from /sys/class/dmi/id/product_uuid in Linux) needs to be unique in each node, otherwise features like VM live migration will be affected. [refer](https://github.com/harvester/harvester/issues/4025)
```

Comment:
```suggestion
- The `product_uuid` fetched from /sys/class/dmi/id/product_uuid in Linux must be unique in each node. Otherwise, features like VM live migration will be affected. For more information, refer to [#4024](https://github.com/harvester/harvester/issues/4025).
```

Reply:
thanks, updated.